pkg/db/v1alpha3/mysql/mysql.go

```diff
@@ -19,7 +19,8 @@ import (
 
 const (
 	dbDriver     = "mysql"
-	dbNameTmpl   = "root:%s@tcp(%s:%s)/%s?timeout=5s"
+	//dbNameTmpl   = "root:%s@tcp(%s:%s)/%s?timeout=5s"
```

Comment:
run `gofmt -w ./pkg` to pass test

Reply:
Thanks a lot!

Reply:
We could remove this line in the future.